### PR TITLE
Prevent infinite energy loop (lava) & balance steam turbine

### PR DESCRIPTION
**Pull request by katecw**
TL;DR:

a) De-duplicated & merged machine{} section of MFR's config
b) MFR's lava fabricator allows infinite power loops so increase its power requirements.
c) Reduce MFR's steam turbine efficiency to bring it more in line with other steam-> power systems.

MFR config general
==================

common.cfg's machine {} section was duplicated. I have merged the two leaving the comments etc. The only ambiguous option was whether the Planter's recipie should be enabled or not (it was in one copy of the section, not in the other). I think the intent of the BR team may have been to disable it since it would have been enabled by default, but the actual effect was the recipie was enabled. I have therefore set it to true.

The planter is also quite useful for automating Pam's crops with golems since some of them don't drop their seeds (eg. soya). Workable around with a use golem but you'd have to select every spot on the field with the golemancer's bell which gets very challenging due to not being able to see because of all the glowy lines.

Balacing power production from lava
===================================

Rationale
---------

The amount of power produced from lava by various machines varies enormously:

 ExU Lava generator: 1B lava -> 40,000 RF

 MFR steam boiler: 400mB/t, 1B lava lasts 800 ticks* = 320,000 mB steam (400mB/tick).
 MFR steam turbine: steam in 80mb/t and output up to 160 RF/t (2 RF/mB steam)
   --> 1B = 640,000 RF

 GT HP lava boiler: 1B lava -> 300,000 mB (32mB/tick)
 GT MV steam turbine: 352 mB/t steam -> 124 EU/t, so 0.382 EU/mB steam (~1.30 RF/mB steam)
   --> 1B lava = 114,462 EU
       1eu ~= 3.4rf, so 1B lava ~= 389,169 RF

       * It is possible to keep the temp just above 100 by slowing the delivery of lava containers, but for these purposes I've assumed max delivery rate.

 GT large heat exchanger: 1B lava -> 160,000 mB steam or 80,000 mB superheated steam
 GT large steam turbine (large titanium rotor): optimal flow 21,000 mB/sec, 1050 EU/t, 130% eff (1,365 EU/t); so 1 mB steam = 1.3 EU/mB steam
   --> 1B lava = 208,000 EU, or ~707,200 RF

 IC2E geothermal generator: 10,000 EU from 1B lava ~= 34,000 RF

In addition, it is possible to make the infamous infinite power loop using the MFR lava fabricator:

 MFR lava fabricator: 1B lava from 200,000 RF

The lava fabricator's config definitely needs to be nerfed to prevent infinite energy loops. However, it seems to me that the order of magnitude difference in power generation from lava also needs addressing. Before working out a suitable energy cost for the lava fab (it should be above the power produced) we need to work that out.

Most people consider making lava a good power source environmentally unfriendly also since it leads to mass destruction of the nether, not to mention being a bit too easy with interdimensional fluid transport devices.

Therefore I propose that we aim to reduce the general lava-power efficiency closer to that of ExU and IC2E. As an aside, ExU doesn't provide configuration options to increase the efficiency of the lava generators either.

We wouldn't really want to mess with the steam to power conversion ratios since that could affect a whole bunch of other things, and for the most part they are well aligned anyway. The exception being the GT large steam turbine which is lower efficiency since it is high-end & high-output.

The real culprits are the GT high pressure lava boiler (a beginning steel age machine) and MFR's steam lava boiler. Both produce ~300k mB steam per bucket of lava. We can't adjust GT's config, and it's super-efficiency is a) well-debated and b) of limited usefulness since it is very slow (32mB steam/tick). You'd need 13 of them to achieve the same steam production rate as one MFR steam boiler.

Proposed changes (in commit)
----------------------------

1) Reduce MFR steam turbine output to to 100 RF/tick (~400k RF per bucket lava with the steam boiler) making it about the same as an HV GT steam turbine.
2) Increase the MFR lava fabricator energy cost to 600k RF/bucket (prevent infinite energy loops).

It is worth considering disabling the lava fabricator completely since it is a form of cheap matter fabrication; you can centrifuge lava for copper, silver, tungstate etc, effectively bypassing UUmatter for those elements, but that is outside the scope of this change.

### CODE REVIEW

**Comment by draknyte1** [comment]
> In addition, it is possible to make the infamous infinite power loop using the MFR lava fabricator:
> 
> MFR lava fabricator: 1B lava from 200,000 RF
> 
> The lava fabricator's config definitely needs to be nerfed to prevent infinite energy loops. However, it seems to me that the order of magnitude difference in power generation from lava also needs addressing. Before working out a suitable energy cost for the lava fab (it should be above the power produced) we need to work that out.

While it does net steam when running lava boilers, the overhead is very minimal.. You'd need about 9-12 Lava boilers, running 6-8 lv turbines, to power 3-4 lava fabs to make enough steam overhead to run a single LV turbine for free.. It's a lot of hassle for very minimal gains.
